main.css

```diff
@@ -1,30 +1,48 @@
 .naugtur .bc-chat-container {
     position: fixed !important;
     z-index: 11511 !important;
-    top: 45px !important;
-    left: 25%;
+    top: 50px !important;
+    right: 200px;
     background: #fff;
-    width: 50% !important;
-    height: calc(96% - 120px) !important;
+    width: 30% !important;
+    height: calc(100% - 110px) !important;
 }
+@media (max-width: 1240px) {
+    .naugtur .bc-chat-container {
+        right: 40px;
+    }
+}
+
+.naugtur #theme-header {
+    display: none !important;
+}
+
+.naugtur #page-wrapper.theme {
+    background: url(http://www.zastavki.com/pictures/originals/2013/Movies___Roger_Pirate_042988_.jpg) top center no-repeat #ddd;
```

Comment:
I got tired of that orange background and got anything out there, but even if it's just a harmless bookmarklet you're right, I'll try to get something on GS's CDN or a free-to-use picture... or even no picture at all, but a gray-ish background at least (enough orange!) xD